client/src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
+import './Hamburger.css';
+
+library.add(faBars, faTimes)
+
+class Hamburger extends Component {
+    constructor(props) {
+        super()
+        this.state = {
+            open: false,
+            iconList: "bars",
+            typeName: "Bars"
+        }
+        this.openMenu = this.openMenu.bind(this)
+    }
+    // let iconList = "bars"
+    // let classList = "";
+    // let classList = "hamburger";
+    
+    // let iconList = `${props.icon}`;
+    
+    // let types = "icon";
+    // let types = "icon";
+    // if (types.includes(props.type)) {
+    //     classList += ` hamburger-${props.type}`
+    // }
```

Comment:
Can you delete commented out code?

Reply:
@ZakkMan , I looked up ways to do a callback prop and tried many ways to try to call it back inside Hamburger.stories.js but because it is not a class component it won't allow me to do it. So then I tried to use function which didn't work either. Next I tried using React Hooks which also is not working. Then I tried to look up how to set a state inside storybook but according to the article Bates and I found its telling us to have two components and place them inside the ./storybook/config.js file, from my understanding is the main file that everyone is using and if we make any changes to it, it will screw up everyones code. Now I'm thinking of going back to class component but if you can specify a little bit more about what you mean when you say "you'll want to pass that back up to the parent. So you'll need a callback prop". I pushed up the code I have with react hooks and commented out my class component code. here is the article I found about setting state inside storybook https://levelup.gitconnected.com/adding-state-to-storybook-in-react-c6744fda25b4